internal/cmd/base/set_rdns.go

```diff
@@ -39,7 +39,7 @@ func (rc *SetRdnsCmd) CobraCommand(s state.State) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringP("hostname", "r", "", "Hostname to set as a reverse DNS PTR entry (required)")
```

Comment:
The docs for this still include "(required)".

Does this need an update to the `Use` line to have both alternatives specified?